frontend/src/pages/NewProject/NewProjectPage.tsx

```diff
@@ -169,12 +169,13 @@ const NewProjectPage = ({ workspace_id }: { workspace_id?: string }) => {
 				}}
 			>
 				<Box
-					marginTop="16"
```

Comment:
So this was part of the position as I saw in the figma design. But I don't think this broke anything?